cookbooks/aws-parallelcluster-platform/spec/unit/recipes/config_login.rb

```diff
@@ -0,0 +1,7 @@
+require_relative '../../spec_helper'
+
+describe 'config_login' do
```

Comment:
the name of this file should be: config_login_spec.rb

Reply:
or better the base recipe should be login_node_config and the spec file: login_node_config_spec.rb